src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -202,6 +202,8 @@ def load_arguments(self, _):
         c.argument('workload_runtime', arg_type=get_enum_type(
             workload_runtimes), default=CONST_WORKLOAD_RUNTIME_OCI_CONTAINER)
         c.argument('snapshot_id', type=str, validator=validate_snapshot_id)
+        c.argument('cluster_snapshot_id', type=str,
+                   validator=validate_cluster_snapshot_id, is_preview=True)
```

Comment:
> type=str

The default value of `type` is `str`, so there is no need to add extra declarations

Reply:
Ok, removed it